### ISSUE TITLE
Scroll problem.

### ISSUE THREAD

**Issue by iapon** [fayeed/dash_chat#10]
Hello, I have a problem with the widget, after I close the keyboard, widget scrolls with the page, see image below
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/6642068/70259465-4a2e2b00-179f-11ea-88ec-4d0fd60b3d31.gif)

How i can beat this behavior?
my code is 
```
  @override
  Widget build(BuildContext context) {
    var appbar = CupertinoNavigationBar(
      leading: CupertinoNavigationBarBackButton(
        onPressed: () => Navigator.of(context).pop(),
        previousPageTitle: 'Chats',
      ),
      middle: Text(
        widget.otherUser.name,
        style: TextStyle(
          color: MyApp.mainColor,
        ),
      ),
      trailing: Icon(
        Icons.more_vert,
        color: Colors.grey,
      ),
    );
    return Scaffold(
      appBar: appbar,
      body: SafeArea(
        child: Container(
          child: DashChat(
            messageContainerFlex: 1,
            height: MediaQuery.of(context).size.height -
                (appbar.preferredSize.height +
                    MediaQuery.of(context).viewPadding.top),
            width: MediaQuery.of(context).size.width,
            user: widget.user,
            inverted: false,
            onSend: (i) {
              setState(() {
                messages.add(i);
                messages.add(
                  ChatMessage(
                    text: "Echo: " + i.text,
                    user: ChatUser(
                      name: widget.otherUser.name,
                      uid: widget.otherUser.uid,
                      avatar: widget.otherUser.avatar,
                    ),
                    createdAt: DateTime.now(),
                  ),
                );
              });
            },
            inputDecoration:
                InputDecoration.collapsed(hintText: "Add message here..."),
            dateFormat: DateFormat('yyyy-MMM-dd'),
            timeFormat: DateFormat('HH:mm'),
            messages: messages,
            showUserAvatar: true,
            showAvatarForEveryMessage: false,
            scrollToBottom: false,
            onPressAvatar: (ChatUser user) {
              print("OnPressAvatar: ${user.name}");
            },
            onLongPressAvatar: (ChatUser user) {
              print("OnLongPressAvatar: ${user.name}");
            },
            inputMaxLines: 3,
            messageContainerPadding: EdgeInsets.only(left: 5.0, right: 5.0),
            alwaysShowSend: true,
            inputTextStyle: TextStyle(fontSize: 16.0),
            inputContainerStyle: BoxDecoration(
              border: Border.all(width: 0.0),
              color: Colors.white,
            ),
            onLoadEarlier: () => {},
          ),
        ),
      ),
    );
  }
```

**Comment by fayeed** [comment]
@iapon Just fixed this you can check the `v1.0.11`, it also includes few more fixes.

**Comment by iapon** [comment]
> @iapon Just fixed this you can check the `v1.0.11`, it also includes few more fixes.

yep, actually fixed
